pytket/extensions/qiskit/backends/ibm.py

```diff
@@ -51,15 +51,20 @@
 )
 
 from pytket.circuit import Circuit, OpType
-from pytket.backends import Backend, CircuitNotRunError, CircuitStatus, ResultHandle
+from pytket.backends import (
```

Comment:
Are you using a different lint / format?

Reply:
Nope. Just using Black. I removed the trailing comma and Black put it back to its original form!